controllers/apply/form-router-next/lib/file-uploads.js

```diff
@@ -19,6 +20,22 @@ const s3 = new AWS.S3({
     region: 'eu-west-2'
 });
 
+const ClamScan = new NodeClam().init({
+    debug_mode: false,
+    scan_recursively: false,
+    clamdscan: {
+        socket: config.get('clamdscan.socket'),
+        timeout: config.get('clamdscan.timeout'),
+        local_fallback: true,
+        path: config.get('clamdscan.path'),
+        config_file: config.get('clamdscan.config_file'),
+        multiscan: false,
```

Comment:
Do we definitely need to set these subsequent params, can we rely on the defaults?

Reply:
Most of these values have tradeoffs to be honest. 

`local_fallback` by default is `false` ie. if local socket fails, it won't fall back to the binary and hence kept it as `true` as we might need a fallback. I'll remove `multiscan` which defaults to multi-thread scanning. I'll remove `reload_db` which by default will **not** look for updated definitions but will be faster. I'll also remove `active` and `bypass_test`.

Reply:
👍